source/reference/minio-mc/mc-batch-generate.rst

```diff
@@ -180,7 +180,8 @@ For the **source deployment**
          | Defaults to ``auto``.
 
```

Comment:
For line 170 - either the `source` or `remote` must have `endpoint: "local"` right? if so i think the comment needs to state *that*. Spitballing:

```
If the source is the :ref:`alias` specified to the command, specify "local" to direct MinIO to use that alias for the endpoint and credentials values. You must omit the `credentials` key for the "local" deployment.
```

Reply:
Maybe also include (for endpoint) to include the URL scheme (as an example you can do something like `Location of the source deployment, such as ``https://minio.example.net``.`

Reply:
Dropping `source`. That is a leftover for when the source deployment had to be local.
And making some additional tweaks to the page related to that.
Otherwise mostly taking this suggestion.